images/iptables/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG IPTABLES_VERSION=1.8.8-1
+
+FROM ubuntu:22.04
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends debian-archive-keyring apt-src && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN echo 'deb-src [signed-by=/usr/share/keyrings/debian-archive-bullseye-automatic.gpg] http://deb.debian.org/debian bookworm main' > /etc/apt/sources.list.d/debian-testing.list && \
```

Comment:
> > sparkles
> > just the docker lint needs to address
> 
> I'm not sure what's next given I've added already a `rm -rf /var/lib/apt/lists/*` to quiet the linter thinking

The linter complains about this line which does an `apt-get update` again. So you probably also need to add `apt-get clean && rm -rf /var/lib/apt/lists/*`.